src/main/java/com/adyen/model/nexo/ICCResetData.java

```diff
@@ -31,12 +33,14 @@
     /**
      * The Atr value.
      */
-    @XmlAttribute(name = "ATRValue")
+    @XmlElement(name = "ATRValue")
+    @Schema(description = "Value of the Answer To Reset of a chip card --Rule: if available", minLength = 1, maxLength = 100)
```

Comment:
Shouldn't this be `minItems` since it's an array?
Or `string` with type `format:byte`?

Reply:
good one, will check with devex if they support `minItems` and create an additional ticket 👍